README.md

```diff
@@ -155,19 +155,18 @@ gcloud auth login
 gcloud config set project $PROJECT_NAME
 
 cd src/app/
-gcloud functions deploy new-dataset-function \
---allow-unauthenticated \
+gcloud functions deploy http-new-dataset-function \
+--no-allow-unauthenticated \
 --gen2 \
---ingress-settings=internal-and-gclb \
+--ingress-settings=all \
```

Comment:
https://cloud.google.com/functions/docs/networking/network-settings
is internal only suitable rather than allowing all